docs/mediapipe.md

```diff
@@ -244,3 +240,27 @@ in the conditions:default section of the deps property:
 ```
 
  Make sure the REGISTER_CALCULATOR(your_calculator); macro is present in the calculator file that you have added.
+
+
+## MediaPipe Graphs Examples <a name="graphs-examples"></a>
+
+[Holistic analysis](../demos/mediapipe/holistic_tracking)
+
+[Image classification](../demos/mediapipe/image_classification/README.md)
+
+[Object detection](../demos/mediapipe/object_detection)
+
+[Multi model](../demos/mediapipe/multi_model_graph/README.md)
+
+
+
+## Current limitations <a name="current-limitations"></a>
+- MediaPipe graphs are supported only for gRPC KServe API.
+
+- KServe ModelMetadata call response contains only input and output names. In the response shapes will be empty and datatypes will be `"INVALID"`.
+
+- Binary inputs are not supported for MediaPipe graphs.
```

Comment:
Well, depends what binary inputs mean here. For passthrough request it's possible. And with Python calc it will also work. So maybe we should mention that binary inputs for OV Tensor type of inputs are not supported?